.vscode/frontend.code-workspace

```diff
@@ -7,7 +7,12 @@
       "path": "../airbyte-webapp-e2e-tests"
     },
     {
-      "path": "../docs"
+      "name": "docs/integrations/destinations",
+      "path": "../docs/integrations/destinations"
+    },
+    {
+      "name": "docs/integrations/sources",
+      "path": "../docs/integrations/sources"
```

Comment:
Are we losing anything by not including all of the docs folder?

Reply:
These are the only parts of the docs that we currently use. Otherwise, other pages will show up in code search.